### PR TITLE
fix: skip values smaller min or greater max

### LINKED ISSUE #2869: Leverage slider shows infinity
If you click on the slider when leverage is at one - it shows -infinity.

Image

### PR DESCRIPTION
**Pull request by holzeis**
fixes #2869

### CODE REVIEW

File: taker-frontend/src/components/Trade.tsx [hunk 1]
```diff
@@ -366,7 +366,12 @@ function Leverage({ leverage_choices, onChange, currentChoice, isLong }: Leverag
                     value={currentChoice}
                     min={min}
                     max={max}
-                    onChange={(val) => onChange(val)}
+                    onChange={(val) => {
+                        if (val < min || val > max) {
+                            return;
```
Comment (Restioson) [inline taker-frontend/src/components/Trade.tsx]: Should it maybe clamp to max?

**Review by bonomat — APPROVED** [review]

File: taker-frontend/src/components/Trade.tsx (hunk 1 above)
Comment (holzeis) [reply taker-frontend/src/components/Trade.tsx]: @Restioson not sure what you mean.. can you elaborate?

Comment (Restioson) [reply taker-frontend/src/components/Trade.tsx]: E.g if val < min then run onChange(min), if val > max then onChange(max)

Comment (holzeis) [reply taker-frontend/src/components/Trade.tsx]: sounds good.. although, it works as is - i'll add your suggestion.

**Review by da-kami — APPROVED** [review]

**Review by klochowicz — APPROVED** [review]

**Pull request merged**